qa/rpc-tests/wallet_changeaddresses.py

```diff
@@ -27,8 +27,7 @@ def setup_network(self):
         args = [
             '-nuparams=5ba81b19:1', # Overwinter
             '-nuparams=76b809bb:1', # Sapling
-            '-txindex',             # Avoid JSONRPC error: No information available about transaction
-            '-experimentalfeatures', '-zmergetoaddress',
+            '-txindex'              # Avoid JSONRPC error: No information available about transaction
```

Comment:
Nit: the `-txindex` line did not need to be modified, as Python allows trailing commas. If you decide to revert this, please squash the change into the original commit.

Reply:
Interesting, i didn't know that :)

If it is my choice i will prefer to don't have the comma even if it is allowed but i suppose that is more a codebase style decision.  I am open to change it if you think it is better.

Reply:
I mentioned it because not modifying that line would make the diff smaller, but didn't block on it because adjusting the commit is probably more effort than a trailing comma is worth 🤷‍♂